### PR TITLE
Sass-mq path change

### PR DESCRIPTION
**Pull request by jbwatson**
Provide a general summary of your changes in the Title above

## Description
Changing sass-mq import to module style

## Related Issue
NA

## Motivation and Context
Why is this change required? 
Blackjack in blackjack container using npm 3 flattens dependencies and the path is not correct.

What problem does it solve?
Any nested blackjack component in sky-pages

What program is it supporting? e.g. Toolkit evolution / Sky Mobile
NA

## How Has This Been Tested?
Please describe in detail how you tested your changes.
We have referenced this branch all the way up through our dependencies and it solves sass-loader errors we were seeing in sky-pages. 

Include details of your testing environment, and the tests you ran to
see how your change affects other areas of the code, etc.
Running sky-pages master locally

## Screenshots (if appropriate)
NA

## Types of Changes
What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Browser Support **Does not apply**
- [ ] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE9
- [ ] IE10
- [ ] IE11
- [ ] Opera
- [ ] Safari
- [ ] Mobile Devices
- [ ] Screen-readers (e.g. JAWS, Apple VoiceOver)

## Checklist **Most of this does not apply**
- [ ] All code includes full inline documentation (as per the [template](https://github.com/sky-uk/toolkit-core/blob/master/_template.scss)).
- [ ] All code conforms to the Toolkit [coding style](https://github.com/sky-uk/toolkit/wiki/Coding-Style).
- [ ] All code conforms to [WCAG 2.0 level AA Accessibility Guidelines](https://www.w3.org/TR/WCAG20/).
- [ ] New functions and mixins have appropriate tests. 
- [x] All new and existing tests passed.
- [ ] I have added instructions on how to test my changes.
- [x] Package version updated.
- [x] CHANGELOG.md updated.

### CODE REVIEW

**Comment by aaronthomas** [comment]
Using the `~` to import as a module is a specific feature of Webpack, and we're trying to keep the Toolkit agnostic to build tools.

You can solve this by adding `node_modules` to your `includePaths` as per Toolkit Usage in the docs: http://www.sky.com/blackjack/toolkit/getting-started

I agree this is a bigger issue though and we do need a better solution moving forward.

**Comment by coderas** [comment]
We've opted to use eyeglass. This is a serious issue for nested components.

**Comment by joebell93** [comment]
Have you tried running this on different projects (other than your nested components)?

Can you provide some detail on how you ran this?

**Comment by jbwatson** [comment]
@joebell93 sorry was late in today and only just seen this. Our structure is:

**sky-pages** imports **component-A** which wraps **component-B**

We changed component-B to use this branch of toolkit-core and then referenced this version in component-A and then imported that component-A into sky-pages. 

Our stuff then built and the rest of sky-pages looked unaffected at a click through. 

**Not worked on toolkit before so how do you guys usually test this stuff?**

**Comment by steveduffin** [comment]
Tested against my [shop-protope app](https://github.com/sky-uk/shop-prototype (uses handlebars / methamsith & shop-toolkit) ), works without issue

**Comment by joebell93** [comment]
Tested:
- Toolkit (`npm run dist`) ✅ 
- Toolbelt app ✅ 
- Sky Pages 🚫 
 - `Module build failed: @import "sass-mq/mq"; ^`

I'll investigate the Sky Pages error further

**Steps before progressing**
- Test some other applications and build tools
- Provide documentation on how eyeglass can be used in the installation instructions

**Comment by jbwatson** [comment]
Sky-pages issue above was tracked down to lack of `node_modules` in their `includePaths`. 

PR raised https://github.com/sky-uk/sky-pages/pull/2301 
Master branch sky-pages builds successfully with this branch and changes from that PR.

**Comment by joebell93** [comment]
Approved https://github.com/sky-uk/sky-pages/pull/2301

Once set-up documentation has been provided and a few more builds have been tested, we can merge

**Comment by aaronthomas** [comment]
This works fine on the Sky Mobile Sales website, I'm happy for this to go ahead.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sky-toolkit-core",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (joebell93) [inline package.json]: Would it be safer to make this `1.2.0` just in case it does break projects?

I'm open for debate

**Pull request merged**